samples/OpenThread/README.md

```diff
@@ -0,0 +1,82 @@
+# 🌶️🌶️ - OpenTHread Networking sample pack
+
+Shows how to use OpenThread Networking API.
+
+## Samples
+
+- [🌶️🌶️🌶️ - UPD OpenThread Client using sockets](UdpThreadClient/)
+- [🌶️🌶️🌶️ - UPD OpenThread Server using sockets](UdpThreadServer/)
+
+Shows how to use various APIs related to OpenThread.
+
+> **Note:** This sample is part of a large collection of nanoFramework feature samples.
+> If you are unfamiliar with Git and GitHub, you can download the entire collection as a
+> [ZIP file](https://github.com/nanoframework/Samples/archive/main.zip), but be
+> sure to unzip everything to access any shared dependencies.
+<!-- For more info on working with the ZIP file, 
+> the samples collection, and GitHub, see [Get the UWP samples from GitHub](https://aka.ms/ovu2uq). 
+> For more samples, see the [Samples portal](https://aka.ms/winsamples) on the Windows Dev Center.  -->
+
+## Hardware requirements
+
+These project are for the ESP32_C6 and ESP32_H2 Espressif devkit boards with a Ws2812B Neopixel on pin 8. 
+This can be easily disabled or Ws2812B added to pin 8 for other boards.
+
+## Sample description
+
+### Upd socket samples
+
+These 2 sample work together to create a client / server communications over OpenThread.
+They use UPD sockets over the IPV6 networking of the OpenThread stack.
+
+The neopixel shows the current role of the node.
```

Comment:
```suggestion
The neopixel shows the current role of the node.

```

Reply:
Changing this doesn't affect MD output as extra line is created after header, just readability in source of MD

Reply:
Well, it does as we are transforming all MD as proper HTML and it's much much better when the Markdown files properly respect the standards. I could have put a linter here in the pipelines but that's fine, I can catch those easilly and can fix easily as well.

The result looks quite cool ;-) It's updated when we do a PR on the docs, so don't search for this sample right away! It's going to be present whenever we'll rebuild the doc. It's then automatic and all links transformed, etc, etc ;-) And that's the only reason I prefer the Markdown to be respectful of the standard.

![image](https://github.com/user-attachments/assets/27a5abc6-2c33-4942-983b-239132e9f77c)